Content.Client/UserInterface/Systems/Sandbox/SandboxUIController.cs

```diff
@@ -109,9 +108,13 @@ public void LoadButton()
 
     private void EnsureWindow()
     {
-        if(_window is { Disposed: false })
+        if (_window is { Disposed: false })
             return;
         _window = UIManager.CreateWindow<SandboxWindow>();
+        // Pre-center the window without forcing it to the center every time.
+        _window.OpenCentered();
+        _window.Close();
```

Comment:
Hm, definitely feels like a proper native way to do this would be nice. This is very much a hack.

Reply:
There's `SetAnchoredPreset` but it attaches the top-left of the window to the anchor point instead of e.g. center-to-center or bottom-to-bottom.

Reply:
I was thinking more like a "Open centered or last position if this isn't the first time" function in the engine, but it's not the biggest problem in the world.